frontend/src/metabase/visualizations/components/Visualization/Visualization-object.unit.spec.js

```diff
@@ -0,0 +1,31 @@
+import { renderWithProviders, screen } from "__support__/ui";
+import Visualization from "metabase/visualizations/components/Visualization";
+import { StringColumn } from "__support__/visualizations";
+
+function setup({ rows, longNameVisibility = "normal" }) {
+  const cols = [
+    StringColumn({ name: "Short name" }),
+    StringColumn({ name: "Long name", visibility_type: longNameVisibility }),
+  ];
+  const series = [
```

Comment:
We shouldn't be doing this. Please use mock data creating helpers like `createMockCard` to always create full data.